[BUG] or [QUESTION] Installation on ARM64

**Describe the bug**
I am using Rosetta2 to install RAiDER on ARM64.

**To Reproduce**

I installed Rosetta2 using `softwareupdate --install-rosetta`.

Following the instructions to use x86 conda packages on ARM64 chips discussed [here](https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon), I found that the package `pybind11` package was required for development install documented [here](https://github.com/dbekaert/RAiDER#5-development).

Without it, I got this error:
```
(RAiDER) --- bekaert-team/RAiDER-forked ‹update_gunw_metadata_json› » pip install -e .
Obtaining file:///Users/cmarshak/bekaert-team/RAiDER-forked
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Preparing editable metadata (pyproject.toml) ... done
Building wheels for collected packages: RAiDER
  Building editable for RAiDER (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building editable for RAiDER (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [104 lines of output]
      /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py:108: _BetaConfiguration: Support for `[tool.setuptools]` in `pyproject.toml` is still *beta*.
        warnings.warn(msg, _BetaConfiguration)
      running editable_wheel
      creating /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER.egg-info
      writing /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER.egg-info/PKG-INFO
      writing dependency_links to /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER.egg-info/dependency_links.txt
      writing entry points to /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER.egg-info/entry_points.txt
      writing top-level names to /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER.egg-info/top_level.txt
      writing manifest file '/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER.egg-info/SOURCES.txt'
      adding license file 'LICENSE'
      adding license file 'LICENSE2.md'
      writing manifest file '/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER.egg-info/SOURCES.txt'
      creating '/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER-0.4.3.dev85+g522bba6.dist-info'
      creating /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-fk_fot9o/.tmp-9roikxn5/RAiDER-0.4.3.dev85+g522bba6.dist-info/WHEEL
      running build_py
      running build_ext
      clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -I/Users/cmarshak/mambaforge/envs/RAiDER/include/python3.10 -c flagcheck.cpp -o flagcheck.o -std=c++17
      building 'RAiDER.makePoints' extension
      creating /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpgme1_x3i.build-temp/tools
      creating /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpgme1_x3i.build-temp/tools/bindings
      creating /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpgme1_x3i.build-temp/tools/bindings/utils
      clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -I/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/numpy/core/include -I/Users/cmarshak/mambaforge/envs/RAiDER/include/python3.10 -c tools/bindings/utils/makePoints.c -o /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpgme1_x3i.build-temp/tools/bindings/utils/makePoints.o
      In file included from tools/bindings/utils/makePoints.c:28:
      /Users/cmarshak/mambaforge/envs/RAiDER/include/python3.10/Python.h:25:10: fatal error: 'stdio.h' file not found
      #include <stdio.h>
               ^~~~~~~~~
      1 error generated.
      Traceback (most recent call last):
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/unixccompiler.py", line 185, in _compile
          self.spawn(compiler_so + cc_args + [src, '-o', obj] + extra_postargs)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/ccompiler.py", line 1041, in spawn
          spawn(cmd, dry_run=self.dry_run, **kwargs)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/spawn.py", line 70, in spawn
          raise DistutilsExecError(
      distutils.errors.DistutilsExecError: command '/Users/cmarshak/mambaforge/envs/RAiDER/bin/clang' failed with exit code 1

      During handling of the above exception, another exception occurred:

      Traceback (most recent call last):
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 147, in run
          self._create_wheel_file(bdist_wheel)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 328, in _create_wheel_file
          files, mapping = self._run_build_commands(dist_name, unpacked, lib, tmp)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 259, in _run_build_commands
          self._run_build_subcommands()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 286, in _run_build_subcommands
          self.run_command(name)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/dist.py", line 1221, in run_command
          super().run_command(command)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/command/build_ext.py", line 84, in run
          _build_ext.run(self)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/Cython/Distutils/old_build_ext.py", line 186, in run
          _build_ext.build_ext.run(self)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 345, in run
          self.build_extensions()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/pybind11/setup_helpers.py", line 290, in build_extensions
          super().build_extensions()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/Cython/Distutils/old_build_ext.py", line 195, in build_extensions
          _build_ext.build_ext.build_extensions(self)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 467, in build_extensions
          self._build_extensions_serial()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 493, in _build_extensions_serial
          self.build_extension(ext)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/command/build_ext.py", line 246, in build_extension
          _build_ext.build_extension(self, ext)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 548, in build_extension
          objects = self.compiler.compile(
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/ccompiler.py", line 600, in compile
          self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/_distutils/unixccompiler.py", line 187, in _compile
          raise CompileError(msg)
      distutils.errors.CompileError: command '/Users/cmarshak/mambaforge/envs/RAiDER/bin/clang' failed with exit code 1
      /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-xl7a1zh7/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py:151: _DebuggingTips: An error happened while installing 'RAiDER' in editable mode.

              ************************************************************************
              The following steps are recommended to help debugging this problem:

              - Try to install the project normally, without using the editable mode.
                Does the error still persists?
                (If it does, try fixing the problem before attempting the editable mode).
              - If you are using binary extensions, make sure you have all OS-level
                dependencies installed (e.g. compilers, toolchains, binary libraries, ...).
              - Try the latest version of setuptools (maybe the error was already fixed).
              - If you (or your project dependencies) are using any setuptools extension
                or customization, make sure they support the editable mode.

              After following the steps above, if the problem still persist and
              you think this is related to how setuptools handles editable installations,
              please submit a reproducible example
              (see https://stackoverflow.com/help/minimal-reproducible-example) to:

                  https://github.com/pypa/setuptools/issues

              More information about editable installs can be found in the docs:

                  https://setuptools.pypa.io/en/latest/userguide/development_mode.html
              ************************************************************************

        _DebuggingTips.warn(project)
      error: command '/Users/cmarshak/mambaforge/envs/RAiDER/bin/clang' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building editable for RAiDER
Failed to build RAiDER
ERROR: Could not build wheels for RAiDER, which is required to install pyproject.toml-based projects
(RAiDER) --- bekaert-team/RAiDER-forked ‹update_gunw_metadata_json› » python -m pip install -e .                                   1 ↵
Obtaining file:///Users/cmarshak/bekaert-team/RAiDER-forked
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Preparing editable metadata (pyproject.toml) ... done
Building wheels for collected packages: RAiDER
  Building editable for RAiDER (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building editable for RAiDER (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [104 lines of output]
      /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/config/pyprojecttoml.py:108: _BetaConfiguration: Support for `[tool.setuptools]` in `pyproject.toml` is still *beta*.
        warnings.warn(msg, _BetaConfiguration)
      running editable_wheel
      creating /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER.egg-info
      writing /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER.egg-info/PKG-INFO
      writing dependency_links to /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER.egg-info/dependency_links.txt
      writing entry points to /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER.egg-info/entry_points.txt
      writing top-level names to /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER.egg-info/top_level.txt
      writing manifest file '/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER.egg-info/SOURCES.txt'
      adding license file 'LICENSE'
      adding license file 'LICENSE2.md'
      writing manifest file '/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER.egg-info/SOURCES.txt'
      creating '/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER-0.4.3.dev85+g522bba6.dist-info'
      creating /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-wheel-htr7d9u_/.tmp-lnb6vanz/RAiDER-0.4.3.dev85+g522bba6.dist-info/WHEEL
      running build_py
      running build_ext
      clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -I/Users/cmarshak/mambaforge/envs/RAiDER/include/python3.10 -c flagcheck.cpp -o flagcheck.o -std=c++17
      building 'RAiDER.makePoints' extension
      creating /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpfxgxrm5x.build-temp/tools
      creating /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpfxgxrm5x.build-temp/tools/bindings
      creating /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpfxgxrm5x.build-temp/tools/bindings/utils
      clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -fPIC -O2 -isystem /Users/cmarshak/mambaforge/envs/RAiDER/include -I/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/numpy/core/include -I/Users/cmarshak/mambaforge/envs/RAiDER/include/python3.10 -c tools/bindings/utils/makePoints.c -o /var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/tmpfxgxrm5x.build-temp/tools/bindings/utils/makePoints.o
      In file included from tools/bindings/utils/makePoints.c:28:
      /Users/cmarshak/mambaforge/envs/RAiDER/include/python3.10/Python.h:25:10: fatal error: 'stdio.h' file not found
      #include <stdio.h>
               ^~~~~~~~~
      1 error generated.
      Traceback (most recent call last):
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/unixccompiler.py", line 185, in _compile
          self.spawn(compiler_so + cc_args + [src, '-o', obj] + extra_postargs)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/ccompiler.py", line 1041, in spawn
          spawn(cmd, dry_run=self.dry_run, **kwargs)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/spawn.py", line 70, in spawn
          raise DistutilsExecError(
      distutils.errors.DistutilsExecError: command '/Users/cmarshak/mambaforge/envs/RAiDER/bin/clang' failed with exit code 1

      During handling of the above exception, another exception occurred:

      Traceback (most recent call last):
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 147, in run
          self._create_wheel_file(bdist_wheel)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 328, in _create_wheel_file
          files, mapping = self._run_build_commands(dist_name, unpacked, lib, tmp)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 259, in _run_build_commands
          self._run_build_subcommands()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py", line 286, in _run_build_subcommands
          self.run_command(name)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/dist.py", line 1221, in run_command
          super().run_command(command)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/command/build_ext.py", line 84, in run
          _build_ext.run(self)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/Cython/Distutils/old_build_ext.py", line 186, in run
          _build_ext.build_ext.run(self)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 345, in run
          self.build_extensions()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/pybind11/setup_helpers.py", line 290, in build_extensions
          super().build_extensions()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/Cython/Distutils/old_build_ext.py", line 195, in build_extensions
          _build_ext.build_ext.build_extensions(self)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 467, in build_extensions
          self._build_extensions_serial()
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 493, in _build_extensions_serial
          self.build_extension(ext)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/command/build_ext.py", line 246, in build_extension
          _build_ext.build_extension(self, ext)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build_ext.py", line 548, in build_extension
          objects = self.compiler.compile(
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/ccompiler.py", line 600, in compile
          self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
        File "/private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/_distutils/unixccompiler.py", line 187, in _compile
          raise CompileError(msg)
      distutils.errors.CompileError: command '/Users/cmarshak/mambaforge/envs/RAiDER/bin/clang' failed with exit code 1
      /private/var/folders/0p/d5x2m4tx5kg1246bplsvyfyh0000gq/T/pip-build-env-cjowilr0/overlay/lib/python3.10/site-packages/setuptools/command/editable_wheel.py:151: _DebuggingTips: An error happened while installing 'RAiDER' in editable mode.

              ************************************************************************
              The following steps are recommended to help debugging this problem:

              - Try to install the project normally, without using the editable mode.
                Does the error still persists?
                (If it does, try fixing the problem before attempting the editable mode).
              - If you are using binary extensions, make sure you have all OS-level
                dependencies installed (e.g. compilers, toolchains, binary libraries, ...).
              - Try the latest version of setuptools (maybe the error was already fixed).
              - If you (or your project dependencies) are using any setuptools extension
                or customization, make sure they support the editable mode.

              After following the steps above, if the problem still persist and
              you think this is related to how setuptools handles editable installations,
              please submit a reproducible example
              (see https://stackoverflow.com/help/minimal-reproducible-example) to:

                  https://github.com/pypa/setuptools/issues

              More information about editable installs can be found in the docs:

                  https://setuptools.pypa.io/en/latest/userguide/development_mode.html
              ************************************************************************

        _DebuggingTips.warn(project)
      error: command '/Users/cmarshak/mambaforge/envs/RAiDER/bin/clang' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building editable for RAiDER
Failed to build RAiDER
ERROR: Could not build wheels for RAiDER, which is required to install pyproject.toml-based projects
```

I ran `python setup.py make` and got:

```
(RAiDER) --- bekaert-team/RAiDER-forked ‹dev➔› » python setup.py make
Traceback (most recent call last):
  File "/Users/cmarshak/bekaert-team/RAiDER-forked/setup.py", line 12, in <module>
    from pybind11.setup_helpers import Pybind11Extension, build_ext
ModuleNotFoundError: No module named 'pybind11'
```

So installed this package and got it to work fine.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,7 +20,7 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 + Verify some constants / equations and remove the comments questioning them
 + Relocate the time resolution of wmodels to one spot
 + Skip test_scenario_3 until a new golden dataset is created
-+ Update scipy requirement
++ Update environment with scipy version minimum and requiring pybind11
 
 ## [0.4.2]
 

--- a/environment.yml
+++ b/environment.yml
@@ -30,6 +30,7 @@ dependencies:
  - numpy
  - pandas
  - progressbar
+ - pybind
  - pydap>3.2.2
  - pyproj>=2.2.0
  - pyyaml

--- a/environment.yml
+++ b/environment.yml
@@ -30,7 +30,7 @@ dependencies:
  - numpy
  - pandas
  - progressbar
- - pybind
+ - pybind11
  - pydap>3.2.2
  - pyproj>=2.2.0
  - pyyaml

